src/main.cpp

```diff
@@ -91,17 +86,17 @@ bool parse_config(char * config_file, server_options *server_options_pointer) {
 
   get_server_options(config, server_options_pointer);
   std::string port = server_options_pointer->port;
-  std::string static_file_root = server_options_pointer->static_file_root;
+  //std::string static_file_root = server_options_pointer->static_file_root;
 
   if (port == "") {
     std::cerr << "Please specify a port.\n";
     return false;
   }
```

Comment:
probably a TODO here to check for proper root paths for the different static handlers